ibis/backends/duckdb/tests/test_udf.py

```diff
@@ -103,3 +104,24 @@ def dont_intercept_null(x: int) -> int:
 )
 def test_dont_intercept_null(con, expr, expected):
     assert con.execute(expr) == expected
+
+
+def test_kwargs_are_forwarded(con):
+    import duckdb
+
+    def nullify_two(x: int) -> int:
+        return None if x == 2 else x
+
+    no_kwargs = udf.scalar.python()(nullify_two)
+    with_kwargs = udf.scalar.python(null_handling="special")(nullify_two)
+
+    # If we return go Non-NULL -> Non-NULL, then passing null_handling="special"
+    # will not change the result
+    assert con.execute(no_kwargs(ibis.literal(1))) == 1
+    assert con.execute(with_kwargs(ibis.literal(1))) == 1
```

Comment:
I can just remove this if you want, this is just to give background on what passing `null_handling="special"` does, but that's not really important.

Reply:
It's fine to keep it.